zepto-dnd.js

```diff
@@ -661,6 +730,9 @@
   }
 
   Sortable.prototype.over = function(e) {
+    if (this.scroller)
+      this.scroller.scrollEvent(this.el, e);
+
```

Comment:
Thanks for your PR! What are these lines for? Is the scroll event bound the the document not enough?